### PR TITLE
Slider component

### PR DESCRIPTION
**Pull request by agustisera1**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://impulsumvc.atlassian.net/browse/COM-141

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Storybook Link** (Link to a local storybook url to review new functionality)

### CODE REVIEW

File: packages/components/src/theme/ComposerTheme/Components/Slider.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  variants: {
+    default: {
+      bg: 'primary-default',
+      sliderBg: 'primary-disabled',
+      boxSize: 3,
+      _focus: {
+        boxShadow: `0px 0px 0px 3px rgba(194, 194, 255, 1)`,
+      },
+    },
+  },
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/Slider.ts:16]: do we need this if there's only 1 variant?

**Review by tkanzakic — COMMENTED** [review]
please, re-export the component inside the `primitives/index.ts` in alphabetical order

File: packages/components/src/theme/ComposerTheme/Components/Slider.ts (hunk 1 above)
Comment (agustisera1) [reply packages/components/src/theme/ComposerTheme/Components/Slider.ts:16]: @tkanzakic I can remove the default variant and use the baseStyle key to try set these styles as alternative. Is it acceptable?

Comment (tkanzakic) [reply packages/components/src/theme/ComposerTheme/Components/Slider.ts:16]: sounds like a better approach to me, what do you think?

Comment (agustisera1) [reply packages/components/src/theme/ComposerTheme/Components/Slider.ts]: @tkanzakic Sounds great! Changes were pushed into the branch.

File: packages/components/src/components/primitives/Slider/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  filledTrackProps,
+  thumbProps,
+  trackProps,
+  ...props
+}) => {
+  const { sliderBg, _focus, boxSize, bg } = useMultiStyleConfig('Slider', {
+    variant: props.variant || 'default',
+  }) as Record<string, RecursiveCSSObject<CSSWithMultiValues>> & {
+    sliderBg: string;
+  };
```
Comment (tkanzakic) [inline packages/components/src/components/primitives/Slider/Slider.tsx:24]: nit: we don't need the variant anymore

**Review by tkanzakic — APPROVED** [review]
just a small comment and LGTM

File: packages/components/src/components/primitives/Slider/Slider.tsx (hunk 2 above)
Comment (agustisera1) [reply packages/components/src/components/primitives/Slider/Slider.tsx:24]: @tkanzakic tweak pushed!

**Review by tkanzakic — APPROVED** [review]
Great work! LGTM!

**Pull request merged**